README.md

```diff
@@ -19,9 +19,50 @@ with `3`, e.g. `pip3` and `python3`, or follow the process for making Python
 
 ## Cloning and running it locally
 
+**Linux deployment:**
+
+Optional installation guides:
+
+Install python3 and pip3 for [linux systems](https://www.tecmint.com/install-pip-in-linux/)
+
+django installation
+`pip3 install django`
+
+Recommended for use with python 3.6 and later
+Install venv for virtual environment (optional)
+`sudo apt install -y python3-venv` - Debian
+Full python3 and venv setup [centOS](https://www.i2tutorials.com/how-to-install-python-set-up-programming-environment-on-centos/)
+
+Recommended for use with python 3.3 and 3.5 deprecated in python 3.6
+Install [pyenv](https://www.tecmint.com/pyenv-install-and-manage-multiple-python-versions-in-linux/) (optional)
+Be sure to install a python3 version in pyenv.
+
+**Run using venv**
+
+1. Clone the project (`git clone https://github.com/codethesaurus/codethesaur.us.git`)
+1. Switch into to directory `cd codethesaur.us`
+1. Use directory as virtual environment `python3 -m venv codethesaur.us`
+1. Activate the directory `source codethesaur.us/activate`
+1. Run `pip install -r requirements.txt`
+1. Then run `python manage.py runserver`
+1. In your browser, visit [http://127.0.0.1:8000/](http://127.0.0.1:8000/) or [http://localhost:8000/](http://localhost:8000/)
+1. Press CTRL+C in the terminal to stop the server.
+
+**Run using pyenv**
```

Comment:
We can probably remove this section too due to removing the above depreciated section.